packages/jest-types/src/Config.ts

```diff
@@ -428,7 +428,7 @@ export type Argv = Arguments<
     onlyFailures: boolean;
     outputFile: string;
     preset: string | null | undefined;
-    projects: Array<string>;
+    projects: Array<string> | Array<Partial<ProjectConfig>>;
```

Comment:
Marking this as a `Partial` since none of the fields in `ProjectConfig` are optional right now, but the docs seem to imply that they are _all_ optional?

It might make more sense to make some/all of the fields in `ProjectConfig` optional instead, but I don't know enough to say which are required and which aren't.